e2e-tests/run-tests

```diff
@@ -14,6 +14,7 @@ MELANGE=${MELANGE:-melange}
 if [ "$MELANGE" = "melange" ]; then
     MELANGE=$(which melange)
 fi
+ARGS=${ARGS:-""}
```

Comment:
This allows to specify Melange extra options, like the runner, which is important for this change.